platform/linux-generic/odp_system_info.c

```diff
@@ -326,6 +327,23 @@ int odp_system_info_init(void)
 
 	fclose(file);
 
+	for (i = 0; i < MAX_CPU_NUMBER; i++) {
```

Comment:
This might be better to be converted to a function, since one might use it to read /sys/devices/system/cpu/cpuX/cpufreq/cpuinfo_cur_freq. There's support for current frequency on some architectures.

Reply:
done